src/moneywiz_api/model/transaction.py

```diff
@@ -402,16 +448,22 @@ def __init__(self, row):
 
         self.original_amount = RDH.get_decimal(row, "ZORIGINALAMOUNT")
         self.original_currency = row["ZORIGINALCURRENCY"]
-        self.recipient_amount = RDH.get_decimal(row, "ZORIGINALRECIPIENTAMOUNT")
+        self.recipient_amount = RDH.get_nullable_decimal(row, "ZORIGINALRECIPIENTAMOUNT")
         self.recipient_currency = row["ZORIGINALRECIPIENTCURRENCY"]
 
         self.original_fee = RDH.get_nullable_decimal(row, "ZORIGINALFEE")
         self.original_fee_currency = row["ZORIGINALFEECURRENCY"]
 
         self.original_exchange_rate = RDH.get_decimal(row, "ZORIGINALEXCHANGERATE")
 
+        self.number_of_shares = RDH.get_nullable_decimal(row, 'ZNUMBEROFSHARES1')
```

Comment:
does `TransferWithdrawTransaction` has  `number_of_shares`?

Reply:
Well, sometimes. But I have just realized that the field `ZNUMBEROFSHARES1` is either NULL or 0, so we may remove this change.